news/2018-03-27-the-qat-gazette-march-2018.md

```diff
@@ -0,0 +1,75 @@
+
+### Creation of the Quality Assurance Helpers (QAH for short)
+The Beatmap Nominators have been internally split with the creation of the Quality Assurance Helpers. With the large influx of newly qualified maps every day, there was no easy way to ensure the quality of all these maps without going back to the old 2014 model of having the small team of QATs checking multiple maps all the time. As a result, the Quality Assurance Helpers was started in order to get Beatmap Nominators more involved in the checking of qualified maps to keep this a community driven effort rather than the opinions of only a few Quality Assurance Team members on every map in qualified. Any full member of the BNG is allowed to join and leave at any time they wish, and activity requirements are fairly low to avoid burn out.
+
+Even if you’re not a Beatmap Nominator, don’t fear because you can still report qualified maps yourself in the [Report a Qualified Beatmap Here!](https://osu.ppy.sh/community/forums/topics/447428) thread, the same as before. The purpose of the Quality Assurance Helpers is to streamline this process and make sure all and not just some maps are checked. Everything else remains the same as before. If you have concerns over a beatmap, post your concerns and it will be looked over by a member of the Quality Assurance Team!
+
+## Within the Quality Assurance Team 
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+People join, people leave. That's how communities evolve throughout history. And the QAT is no different. 
+### New Quality Assurance Team additions
+Thanks to the QAH initiative explained above, we have determined some of our Beatmap Nominators were qualified enough to become part of our team. With the addition of the new branches explained below, each one of the new QAT members is assigned to one or various branches, making our duties more organized. 
+
+Please give a warm welcome to:
+- [-Mo-](https://osu.ppy.sh/users/2202163) and [Mir](https://osu.ppy.sh/users/8688812), who will be joining the **Disqualification** branch;
+- [Gabe](https://osu.ppy.sh/users/654108), who will partake the **Moderation** branch; 
+- and finally, [Lanturn](https://osu.ppy.sh/users/1446665), who will assist the **Metadata** branch.
+### Quality Assurance Team retirements
+Not everything can always be good news, for it is at times of loss when one really appreciates what one had
+
+- After 5 years of service and countless efforts made for the team, being both in the BAT and the QAT, the judge of mapping contests, co-creator of Beatmap Nominator tests, our long-lasting member [Irreversible](https://osu.ppy.sh/users/1287964) has made the decision of working on his own. He will still hold the Olympiad contests, though, so stay alert!
```

Comment:
I have mentioned the Olympiad in some previous publications, but clarifying that it's a successor to the MBC will probably be more familiar to most people.